### PR TITLE
fixed messenger issues

### PR DESCRIPTION
**Pull request by NEUDitao**
the db stuff was causing some issues, this *should* fix them

### CODE REVIEW

File: backend/database.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ import MockFirebaseRef from './MockFirebaseRef';
 class Database {
   constructor() {
     if (macros.PROD) {
+      this.db = this.loadDatabase();
```
Comment (dajinchu) [inline backend/database.js]: why load twice?

**Comment by dajinchu** [comment]
what were the problems?

File: backend/database.js (hunk 1 above)
Comment (NEUDitao) [reply backend/database.js]: outdated :P

**Comment by NEUDitao** [comment]
Issues were:

- I think the dev db stores the db as a list, so we were doing `const user of users`, which was breaking, since `users` was an object from Firebase, and objects aren't iterable. Fixed with an `Objects.values(users)`

- user.js would freak out when watchingSections and watchingClasses didn't exist, which happens when a user unsubscribes using STOP on the messenger bot

**Comment by dajinchu** [comment]
have you tested this on real firebase data?

**Comment by NEUDitao** [comment]
yeah, i had the db pointed to prod when I was testing this (explaining the double load)

**Review by ryanhugh — APPROVED** [review]

**Comment by ryanhugh** [comment]
Biggest problem: 
 - If you go to the site right now and try to use the notification feature that Eddy just built, the site crashes and you can't sign up for notifications.

**Pull request merged**